package.json

```diff
@@ -26,7 +26,7 @@
     "posttest": "istanbul check-coverage"
   },
   "dependencies": {
-    "harvest": "git+ssh://git@github.com:holidayextras/harvest.git#v2.0.0",
+    "harvest": "git+ssh://git@github.com:holidayextras/harvest.git#WEB-11095",
```

Comment:
did you forget it, or you're gonna change it later to `git+ssh://git@github.com:holidayextras/harvest.git#2.0.1`